### DISCUSSION TITLE
come si crea la directory `.github/workflows/gh-deploy.yml` per far compilare il progetto su pagine web di Github

### DISCUSSION THREAD

**Discussion by cirospat** [opendatasicilia/ods-mkdocs-material#15] [Domande]
Al fine di visualizzare il sito creato con **MKdocs-material** su pagine web di Github, è necessario generare la directory:
**`.github`**
allinterno della quale c'è la directory:
`workflows`
all'interno della quale c'è il file:
`gh-deploy.yml`

C'è una modalità per far generare a Github quanto sopra indicato.
Nelle risposte che seguono viene descritta la modalità.

**Comment by aborruso** [comment]
Caro @cirospat si crea tutto a mano: crei le due cartelle e dentro crei il file di deploy.

Quello che trovi in questo repo, per come è fatto, attiverà la compilazione e pubblicazione automatica del sito a ogni push del codice.

**Comment by gbvitrano** [comment]
@cirospat ho aperto un issue per gh-pages #22 appena posso la completo
